spyder/plugins/ipythonconsole/widgets/namespacebrowser.py

```diff
@@ -101,6 +101,8 @@ def get_value(self, name):
             raise ValueError(msg % reason_big)
         except UnpicklingError:
```

Comment:
Since both `PicklingError` and `UnpicklingError` raise the same error message, you can join them here, like this:

```python
except (PicklingError, UnpicklingError):
    raise ValueError(msg % reason_not_picklable)
```